sections/resources/uas/UASLayout.jsx

```diff
@@ -17,7 +17,7 @@ export const UASLayout = () => (
         {
           color: "yellow",
           text: "Join our Community of 800+ Students",
-          link: "/resources/uas" /* TODO: change to Discord server invite link */,
+          link: "/resources/" /* TODO: change to Discord server invite link */,
```

Comment:
Can we remove the buttons from UAS Layout? I don't think this is necessary since we have the button again later on the page